src/js/components/NavTabsComponent.jsx

```diff
@@ -24,8 +41,8 @@ var NavTabsComponent = React.createClass({
         "active": tab.id === activeTabId
       });
 
-      var badge = tab.badge > 0 ?
-        <span className="badge">{tab.badge}</span> :
+      var badge = tab.id === "deployments" && this.state.activeDeployments > 0 ?
```

Comment:
Now I would prefer an own function -> getBadge() , it got a little bit more complicated now.